projects/storefrontlib/shared/test/mock-feature-level-directive.ts

```diff
@@ -16,6 +16,8 @@ export class MockFeatureLevelDirective {
   ) {}
 
   @Input() set cxFeatureLevel(_feature: string | number) {
-    this.viewContainer.createEmbeddedView(this.templateRef);
+    if (!_feature.toString().includes('!')) {
```

Comment:
This is a really interesting condition you've added. Perhaps you could leave a comment explaining what it is for? It is a widely used directive so we need to be careful about changing anything here.

Reply:
Added comment for clarity